content/blog/en/dojo-middleware.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Custom Dojo Middleware
+date: 2020-02-03
+author: Anthony Gubler & Rene Rubalcava
+---
+
+Dojo provides a [middleware system](https://dojo.io/learn/middleware/introduction) that you can use in developing widgets for your applications. There is a comprehensive list of [available middleware](https://dojo.io/learn/middleware/available-middleware) that you can use to manage local widget state, styling, or DOM related information.
+
+<!-- more -->
+
+Middleware is really interesting because they can be used to help you interact with the DOM or with the properties of your widget.
```

Comment:
```suggestion
Middleware are really interesting because they can help you interact with the DOM or with the properties of your widget with retaining a reactive architecture.
```